### ISSUE TITLE
How to turn on/off the Hvac Auto?

### ISSUE THREAD

**Issue by Whirlwind** [teslamotors/vehicle-command#283]
Hello, I can not find the API in the protobuf file about turn on/off the Hvac Auto.

By the way, I find the `HvacTemperatureAdjustmentAction`:
https://github.com/teslamotors/vehicle-command/blob/eaf6ee1fee2bcc2bcbb44dfea921e9fab78ef249/pkg/protocol/protobuf/car_server.proto#L202-L226

How to use it? I send the data like this:

```
HvacTemperatureAdjustmentAction {
    absolute_celsius: 22,
    hvac_temperature_zone : {
        TEMP_ZONE_FRONT_LEFT: {}
    }
}
```

It seams that the `absolute_celsius` not works, because the `driver_temp_celsius` always be 0, so the Vehicle will change the temperature to `LOW`.

**Comment by beachmiles** [comment]
Yes there are many API commands that would be nice to have like turning HVAC to manual or auto mode. Wish we could add some very basic HVAC features like a heater off mode and a vent mode. Someone just wrongly closed my request thinking it's already implemented.

The cabin heater still wrongly is being used when cooling is only desired.
https://github.com/teslamotors/vehicle-command/issues/112
